ui/src/actions/login.js

```diff
@@ -149,3 +149,20 @@ export function doSignOut(navigate) {
     });
   };
 }
+
+export function signOutUser(username, navigate) {
```

Comment:
There is already a [signout](https://github.com/mxcube/mxcubeweb/blob/81381c34c11f802d129e993c094e3bd267d9b74f/ui/src/actions/remoteAccess.js#L102), did you try to this one or did it not do what you liked ?

Reply:
This [signout](https://github.com/mxcube/mxcubeweb/blob/81381c34c11f802d129e993c094e3bd267d9b74f/ui/src/actions/remoteAccess.js#L102) works when you are already logged in. In our case we are not yet logged in.